ouroboros-network/src/Ouroboros/Network/TxSubmission/Inbound.hs

```diff
@@ -227,6 +229,16 @@ txSubmissionInbound _tracer maxUnacked mpReader mpWriter =
                 -> Collect txid tx
                 -> m (ServerStIdle n txid tx m ())
     handleReply n st (CollectTxIds reqNo txids) = do
+
+      -- Check they didn't send more than we asked for. We don't need to check
+      -- for a minimum: the blocking case checks for non-zero elsewhere, and
+      -- for the blocking case it's quite normal for them to send us none.
```

Comment:
I think you mean: `and for the non-blocking case it's quite normal for them to send us none.`

Reply:
Well spotted thanks.